From 3336348a86581a91f2d98efbbcef889f46d251cd Mon Sep 17 00:00:00 2001
From: Ivan Iziumov <iiziumov@anaconda.com>
Date: Fri, 16 Jan 2026 16:32:44 +0100
Subject: [PATCH] fix problematic gtests

Rebased onto 1.30.0: upstream dropped the TensorRT-only COMPILE_FLAGS block after AddTest; only the AddTest call remains to disable.
---
 cmake/onnxruntime_unittests.cmake | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/cmake/onnxruntime_unittests.cmake b/cmake/onnxruntime_unittests.cmake
index 1974b5d..7b39f25 100644
--- a/cmake/onnxruntime_unittests.cmake
+++ b/cmake/onnxruntime_unittests.cmake
@@ -2851,12 +2851,12 @@ if (onnxruntime_BUILD_SHARED_LIB AND NOT CMAKE_SYSTEM_NAME STREQUAL "Emscripten"
     list(APPEND onnxruntime_ep_graph_test_LIBS ${TENSORRT_LIBRARY_INFER})
   endif()
 
-  AddTest(DYN
-          TARGET onnxruntime_ep_graph_test
-          SOURCES ${onnxruntime_ep_graph_test_SRC} ${onnxruntime_unittest_main_src}
-          LIBS ${onnxruntime_ep_graph_test_LIBS}
-          DEPENDS ${all_dependencies}
-  )
+  #AddTest(DYN
+  #        TARGET onnxruntime_ep_graph_test
+  #        SOURCES ${onnxruntime_ep_graph_test_SRC} ${onnxruntime_unittest_main_src}
+  #        LIBS ${onnxruntime_ep_graph_test_LIBS}
+  #        DEPENDS ${all_dependencies}
+  #)
 endif()
 
 include(onnxruntime_fuzz_test.cmake)
